Define SUN_LEN.

It's missing in our headers, so add it here.

diff -ur spice-orig/server/reds.c spice/server/reds.c
--- spice-orig/server/reds.c	2017-07-31 21:10:22.747216004 +0200
+++ spice/server/reds.c	2017-07-31 21:17:50.342461769 +0200
@@ -93,6 +93,9 @@
 
 static TicketAuthentication taTicket;
 
+#define SUN_LEN(ptr) ((size_t) (((struct sockaddr_un *) 0)->sun_path)	      \
+		      + strlen ((ptr)->sun_path))
+
 static int spice_port = -1;
 static int spice_secure_port = -1;
 static int spice_listen_socket_fd = -1;
